eng/common/templates/job/performance.yml

```diff
@@ -71,11 +71,11 @@ jobs:
         # Run all of the steps to setup repo
         - ${{ each step in parameters.steps }}:
           - ${{ step }}
-        - powershell: $(Build.SourcesDirectory)\eng\common\performance\performance-setup.ps1 $(IsInternal) -Framework $(_Framework) ${{ parameters.extraSetupParameters }}
+        - powershell: $(Build.SourcesDirectory)\eng\testing\performance\performance-setup.ps1 $(IsInternal) -Framework $(_Framework) ${{ parameters.extraSetupParameters }}
```

Comment:
These changes should be done in arcade as these yml templates are under common. 

It feels weird that common yml templates reference a folder outside of common.

Reply:
Turns out `performance.yml` is actually dead code. I moved the one we are still using (and will make an Arcade PR to delete them all.)